kubeadm/kube-proxy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG k8sVersion="1.17.3"
```

Comment:
[3]

Reply:
Yeah this is just the default, and so can be overridden with `docker build --build-arg k8sVersion="1.18.0"`. We'll need to set up some automation to produce new images on new releases though.

Reply:
i've been playing with GitHub actions in this regard so automation seems possible.

but it this goes back to the topic of whether we want to branch / tag this repository and or have the install scripts in folders.

i think this is always the safest bet, because otherwise if a single set of scripts and docker files is maintained for the whole support skew of k8s it might end up with some `if version ...` branches.

Reply:
What do you think of using github releases for the script and manifest artifacts? I don't want to create version subdirectories since they are going to be 99% identical to each other.

Reply:
having releases / tags is fine, but without branches it will not allow backports.
one option is to start with a linear tag increments in a single branch (master), but if a change is required for a backport a branch can be created.